THIRD-PARTY-LICENSES.csv

```diff
@@ -50,18 +50,18 @@
 "@babel/helper-function-name@7.19.0","MIT","https://github.com/babel/babel"
 "@babel/helper-hoist-variables@7.18.6","MIT","https://github.com/babel/babel"
 "@babel/helper-member-expression-to-functions@7.18.9","MIT","https://github.com/babel/babel"
-"@babel/helper-module-imports@7.18.6","MIT","https://github.com/babel/babel"
```

Comment:
Is it correct that the 226 lines/dependencies are added due to that odd compilation error?

Reply:
Ran `npm install` again to be sure; I'm afraid so (actually went up two).

![image](https://github.com/dockstore/dockstore-ui2/assets/1049340/793a2451-4d34-40c3-9d10-257e30c8e2d3)

Reply:
Idle thought, possibly spin-off ticket. Shouldn't the third party licenses file just be a list of dependencies that end up being part of the build as opposed to dev dependencies? 
i.e. in the old days, I would have thought that if I had like a paid-for licensed Intel C++ compiler that compiled my program, I wouldn't necessarily include that in a list of third party licenses. Is this similar?

Reply:
> Idle thought, possibly spin-off ticket. Shouldn't the third party licenses file just be a list of dependencies that end up being part of the build as opposed to dev dependencies? i.e. in the old days, I would have thought that if I had like a paid-for licensed Intel C++ compiler that compiled my program, I wouldn't necessarily include that in a list of third party licenses. Is this similar?

Hmm, not sure, I poked around, but it's not clear to me; we might need to ask a lawyer. The [license checker](https://www.npmjs.com/package/license-checker) we use has a production only option, so we could exclude the dev dependencies if we wanted to. I'll create a follow-up ticket.

Reply:
dockstore/dockstore#5549